From: Karolina Surma <33810531+befeleme@users.noreply.github.com>
Date: Thu, 5 Aug 2021 19:17:16 +0200
Subject: Fix test_other_with_assets_new_style w/Sphinx 4.0 (#133)

Make the test work with Sphinx 4.0.x.
Fix the typo so that the regression file is actually found.

(cherry picked from commit 0f185f426caad91c2cd18229ca3d458b858b396e)
---
 tests/test_build.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/test_build.py b/tests/test_build.py
index ee01aa9..37a63bd 100644
--- a/tests/test_build.py
+++ b/tests/test_build.py
@@ -64,9 +64,9 @@ def test_other_with_assets(app, check_asset_links):
 
 @pytest.mark.sphinx(testroot="linenos")
 @pytest.mark.skipif(
-    sphinx.version_info[:2] <= (4, 0), reason="Test uses Sphinx 4 code blocks"
+    sphinx.version_info[:2] < (4, 0), reason="Test uses Sphinx 4 code blocks"
 )
-def test_other_With_assets_new_style(app, check_asset_links):
+def test_other_with_assets_new_style(app, check_asset_links):
     check_asset_links(app)
 
 
